biochemist-python/chapters/Working_with_Pandas.ipynb

```diff
@@ -120,6 +120,7 @@
       "File_Parsing.ipynb\r\n",
```

Comment:
<p>Should I also move the <code>ls data</code> command into a markdown cell? I added this statement to the markdown cell above the 'ls data' command.</p><p> </p><p>Please note that when you use the <code>ls data</code> command, the list of files and folders that appears may be different from the list shown here. It just depends on which files are located on your computer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MolSSI-Education/python-scripting-biochemistry/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>